canonical/vanilla-framework
# Use new documentation layout on Vanilla site

## Done

Apply new documentation layout to Vanilla website.
This mostly affects docs section, but the whole site now uses paper background as well.

Fixes [WD-6055](https://warthogs.atlassian.net/browse/WD-6055)

## QA

- Open [demo](https://vanilla-framework-4863.demos.haus/docs)
- Review the docs, make sure layout is fine and components look good on new background



[WD-6055]: https://warthogs.atlassian.net/browse/WD-6055?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
